### ISSUE TITLE
Unit test coverage displays 0%

### ISSUE THREAD

**Issue by adityacs** [qax-os/goreporter#73]
goreporter displays test coverage as 0%. However, I have few unit tests and running with
`go test -cover` gives me proper coverage %. 

Kindly help here.

**Comment by wgliang** [comment]
Can you post the logs? Or any related information.

**Comment by adityacs** [comment]
goreporter html
![image](https://user-images.githubusercontent.com/7681197/48121096-074b0c80-e29a-11e8-8c0a-bb27bf717aea.png)

I am using below command to run goreporter
`goreporter -p . -r . -f html`

console output
![image](https://user-images.githubusercontent.com/7681197/48123219-5f384200-e29f-11e8-9f02-11564c415942.png)

**Comment by wgliang** [comment]
Yeah, you can see log that the tool can not find your package db on your $GOPATH.

**Comment by adityacs** [comment]
@wgliang  What's the correct command to run goreporter?
 I have "db" package in "C:\Users\csad\git\vzt-poseidon\src\bitbucket.vtitel.net\vzt-poseidon\db" and goreporter is searching in all wrong places.

Also, GOPATH is set correctly. Still goreporter is looking in wrong folders

**Comment by wgliang** [comment]
you can just run your command like
`goreporter -p ./ -r ./ -f html`

goreporter will search your packages on your $GOPATH, and it also support vendor. Obviously the path `C:\Users\csad\git\vzt-poseidon\src\bitbucket.vtitel.net\vzt-poseidon\db` is not in your GOPATH(`C:\Users\go`).

**Comment by wgliang** [comment]
I think you can just move your repo into `C:\Users\go\src`. And then run the command again.

**Comment by adityacs** [comment]
I have set the GOPATH to `C:\Users\csad\git\vzt-poseidon`. Still I get same error
